### PR TITLE
Update Cadence v0.26.0

### PR DESCRIPTION
**Pull request by turbolent**
## Description

- Update to [Cadence v0.26.0](https://github.com/onflow/cadence/releases/tag/v0.26.0)
- Update to [Go SDK v0.27.0](https://github.com/onflow/flow-go-sdk/releases/tag/v0.27.0)
- Update to latest Flow Go (currently PR https://github.com/onflow/flow-go/pull/3077, will update to latest commit of `master` once merged)
- https://github.com/onflow/flow-go/pull/2981 changed the state holder / state view API, adjust it accordingly

## Problems

`TestSubmitTransactionWithCustomLogger` fails with:

```
{"level":"debug","error":"[Error Code: 1112] could not get execution parameter from the state (address: f8d6e0586b0a20c7 path: storage/executionEffortWeights)","message":"could not set execution effort weights. Using defaults"}
```

@janezpodhostnik Do you know how the test must be adjusted? Does the emulator need to initialize the execution state in some way?

______

For contributor use:

- [x] Targeted PR against `master` branch
- [x] Linked to GitHub issue with discussion and accepted design OR link to spec that describes this work
- [x] Code follows the [standards mentioned here](https://github.com/onflow/flow-emulator/blob/master/CONTRIBUTING.md#styleguides)
- [x] Updated relevant documentation
- [x] Re-reviewed `Files changed` in the GitHub PR explorer
- [x] Added appropriate labels

### CODE REVIEW

File: blockchain.go [hunk 1]
```diff
@@ -1015,30 +1015,31 @@ func (b *Blockchain) commitBlock() (*flowgo.Block, error) {
 func (b *Blockchain) GetAccountStorage(address sdk.Address) (*AccountStorage, error) {
 	program := programs.NewEmptyPrograms()
 
-	env, err := fvm.NewTransactionEnvironment(
+	meterParameters := meter.DefaultParameters().
```
Comment (pattyshack) [inline blockchain.go:1018]: you can skip this.  the default value is already max int64

**Review by pattyshack — APPROVED** [review]

File: blockchain.go (hunk 1 above)
Comment (turbolent) [reply blockchain.go]: Nice. Simplified it in [37fac23](https://github.com/onflow/flow-emulator/pull/194/commits/37fac2343d3b38b3ff79032cd436b9f624c738a7)

**Review by sideninja — APPROVED** [review]

**Comment by sideninja** [comment]
@turbolent Thank you! looks good but take a look at the test error and make sure that gets resolved.

**Comment by turbolent** [comment]
@janezpodhostnik thank you for fixing the test case! 🙏

**Pull request merged**